pages/api/Schema/reports/financialAccounts/financialAccounts/financialAccounts.graphql

```diff
@@ -0,0 +1,35 @@
+extend type Query {
+  financialAccountSummary(
+    input: FinancialAccountSummaryInput!
+  ): [FinancialAccountSummaryResponse]!
+}
+
+input FinancialAccountSummaryInput {
+  accountListId: ID!
+  financialAccountId: ID!
+}
+
+type FinancialAccountSummaryResponse {
+  id: ID!
+  closingBalance: String
+  credits: String
+  debits: String
+  difference: String
+  endDate: String!
+  openingBalance: String
```

Comment:
Should these numbers be `Float`s instead? Or are we using a `String` because it is a string in the REST API?

Reply:
They return from the REST API as strings